theme/src/components/PageFlipper/useArrowNavigation.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react'
+
+export default ({ nextPage, previousPage }, onNavigate) => {
+  const handleKeyStroke = e => {
+    if (e.keyCode === 37 && previousPage) {
+      onNavigate(previousPage)
+    }
+
+    if (e.keyCode === 39 && nextPage) {
+      onNavigate(nextPage)
+    }
+  }
+
+  useEffect(() => {
+    window.addEventListener("keydown", handleKeyStroke)
+
+    return () => {
+      window.removeEventListener("keydown", handleKeyStroke)
+    }
+  })
```

Comment:
Should we add an empty array here? 
So that this happens only happens onMount, not on every render?

Reply:
Yeah, that makes sense. I tried adding it in useSwipeNavigtion, but there everything stopped working, which I don't really understand.